### ISSUE TITLE
Traffic lights are not easy for people with a vision issue. 

### ISSUE THREAD

**Issue by weallhave24hrs** [Ambridge/Ambridge#29]
Concern express to me by a person who is color blind.  The light poles were all painted the same color and for a person with a color issue the poles color matters. 

Asked them to tell boro but unsure if it was done.

**Comment by whit537** [comment]
Interesting. I've heard of the orientation of the lights mattering to people with red/green color blindness (when the lights are vertical the order is clear: red at the top, green at the bottom; but when the lights are sideways, is red on the left or right?), but I haven't heard before that the *pole* color matters. Is this something you've encountered before, @jmkauer?

**Comment by jmkauer** [comment]
Good Morning!  I am not aware of an issue relative to color blindness as it relates to the color of the signal post.   The important aspect of the signal is actually the signal itself and truly not the post that it's affixed to.  The Borough has actually started a dialogue with State Representative Matzie to partner with the Commonwealth to advance an effort to replace and upgrade all of the traffic signals, posts, and controllers along State Route 989 in the Borough.   In the interim we are submitting a grant application next month to PENNDOT to upgrade the remaining signals on Duss Avenue to LED lighting.  This will improve the sustainability of the signal operations and visibility.  In regards to the many black posts and supports in Ambridge, I think they are aesthetically appropriate and add to the overall streetscape in the Borough; however, I'd be interested in learning more about how people with color blindness relate to traffic signals.  

Joe Kauer, Ambridge Borough Manager

**Comment by whit537** [comment]
> I'd be interested in learning more about how people with color blindness relate to traffic signals.

Likewise!

@weallhave24hrs Any chance we can get the original person who raised this to contribute directly on this ticket?

**Comment by BergFulton** [comment]
Tangentially- something I've noticed as a runner and dog walker, not only in the neighborhood but also in the city where I work, that traffic lights with and without walk signals are barely long enough for able bodied people to cross with sufficient time, let alone someone with mobility impairment. It might be interesting to gather some data about light timing vs. pedestrian crossing ability.

**Comment by whit537** [comment]
In lieu of our color-blind neighbor sharing their experience with us ... I found this blog post, "[Colorblind at the Traffic Light](http://www.color-blindness.com/2007/02/06/colorblind-at-the-traffic-light/)." Doesn't mention pole color one way or another.

**Comment by whit537** [comment]
> gather some data about light timing vs. pedestrian crossing ability

Reticketed as #30. :)

**Comment by Rdonnelly45** [comment]
I was at a meeting recently with @weallhave24hrs and we both heard about the issue further straight from the colorblind person who brought this up originally. It is the black light housings (and posts) that are causing the problem at night specifically. This colorblind person uses the position of the light in the housing to know if it's the red or green one that's currently lit. Top=red=stop and bottom=green=go. I know the article @whit537 posted above claims "but that's wrong," however this is what we were told by the person having the issue. Yellow is the preferred color for the light housings and poles, so what is surrounding the lights can be seen at night. I can attest that the black poles and housings are impossible to see at night on some streets, such as Duss Ave. for example when I was headed south at night right after this meeting. It is unknown at this point if PennDOT has any rules about the color that light housings and poles should be.

**Comment by Rdonnelly45** [comment]
I should also note, even for those of us with no color issues, I suppose black light housings/poles are not ideal because what if a bulb goes out (or power goes out). Being able to see that an unlit traffic signal is there at night will help everyone drive safer.

**Comment by whit537** [comment]
Ah, that makes a lot of sense @Rdonnelly45. Of course it'd be hard to judge the relative position of the light that's on, if it appears to be just floating in space with no fixed reference point.

**Comment by dmk246** [comment]
I know in certain towns such as Beaver, they have the signals for walk and don't walk but they also have a noise associated with each one as well.  I was told that was put in place for the blind .... so they could utilize the noise as well as any other ways to verify it is safe to cross. Just thought I'd mention that ... I always thought that was a good idea...

**Comment by whit537** [comment]
Good call, @dmk246. Reticketed as https://github.com/Ambridge/Ambridge/issues/62. :-)
